CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [1.8.5] - 2015-09-28
+- Saving "world_readable" to your config file via `plotly.tools.set_config` actually works.
+- You can also save `auto_open` and `sharing` to the config file so that you can forget these
+  keyword argument in `py.iplot` and `py.plot`.
+
+
```

Comment:
oops, after praising the changelog, i totally didn't add a note! gah!

Also, this should now be 1.8.6, the update hit at 1.8.5, mind giving it a quick change?